### ISSUE TITLE
Album cover as lockscreen background

### ISSUE THREAD

**Issue by ghost** [kienvtqhi/melocloud-issue-tracker#21]
Other players can change the lockscreen background to the album cover of the currently playing track. It would be nice to add this feature (as an option) to melocloud

**Comment by namhoang159** [comment]
Actually this feature is available on Android OS < 4.4
Did you try this feature with other players on your current device?

**Comment by ghost** [comment]
Yes, I tried with shuttle player and phonograph. Both have this option in the settings. I'm running 6.0.1 on a LG Nexus 5.

**Comment by namhoang159** [comment]
Okay, we will consider this issue. 
Thanks!
